components/SlidingMenu.vue

```diff
@@ -64,18 +64,22 @@
     </div>
 
     <div class="sliding-menu__buttons">
-      <v-btn class="btn--likecoin" flat icon large color="primary">
+      <v-btn class="btn--likecoin btn--search" flat icon large color="primary">
```

Comment:
I only spot this combination is non-intuitive to me. Not sure why we have two btn class here.